sourcecode/hub/resources/views/user/my-account.blade.php

```diff
@@ -0,0 +1,25 @@
+<x-layout>
+    <x-slot name="title">{{ trans('messages.my-account') }}</x-slot>
+
+    <h6>{{ trans('messages.change-profile-name') }}</h6>
+    @if(session('success'))
+        <div class="alert alert-success" role="alert" id="success-alert">
+            {{ session('success') }}
+        </div>
+    @endif
+
+    <x-form action="{{ route('user.update-username') }}" method="POST">
```

Comment:
`method="POST"` and `@csrf` aren't necessary, the form component has those by default.